src/server/img_storage.rs

```diff
@@ -199,11 +393,21 @@ impl StoreImage {
         )
         .await
         {
-            Ok(v) => Ok(StoreResult {
-                hash: v.etag.clone(),
-                url: public_url.parse()?,
-                object: v,
-            }),
+            Ok(mut object) => {
+                object.content_disposition = Some("inline".to_owned());
+                object.cache_control = Some(format!("public, max-age={}", self.settings.cache_ttl));
+                object.update().await.map_err(|_| {
+                    error!("Could not set disposition for {:?}", object.self_link);
+                    HandlerErrorKind::BadImage("Could not set content disposition")
```

Comment:
Nothing to hold this up over but I think some of the Internal and BadImage errors in here derived from `cloud_storage::Error` should have their own `HandlerKind` `CloudStorage(#[from] cloud_storage::Error)` variant

Reply:
Yeah, I figure we can revisit and revise the errors later, probably when we break things up to componentize stuff.  For now, having the one, semi-universal errors is "good enough".